packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -1,25 +1,44 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { addDays, daysFromTodayTo, toMoment, convertDateFormat, getPosition, isMobile } from '@deriv/shared';
 
-import Input from './date-picker-input.jsx';
-import Calendar from './date-picker-calendar.jsx';
-import Native from './date-picker-native.jsx';
+import Input from './date-picker-input';
+import Calendar from './date-picker-calendar';
+import Native from './date-picker-native';
 import MobileWrapper from '../mobile-wrapper';
 import DesktopWrapper from '../desktop-wrapper';
 import { useOnClickOutside } from '../../hooks/use-onclickoutside';
 
-const DatePicker = React.memo(props => {
+type TDatePickerOnChangeEvent = {
+    date?: string;
+    duration?: number | null | string;
+    target?: { name: string; value: number | string | moment.Moment | null };
+};
+type TDatePicker = Omit<
+    React.ComponentProps<typeof Native> & React.ComponentProps<typeof Input> & React.ComponentProps<typeof Calendar>,
+    'value'
```

Comment:
```suggestion
    'value' | 'onSelect'
```
@amir-deriv as I can see, onSelect is not passed to DatePicker as a prop, so we need to omit it too.